### PR TITLE
add makeflyeasy PixPilot-V6

### PR DESCRIPTION
**Pull request by Cedric0489**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@Cedric0489 need to add to common-autopilots.rst   also, if the power module is NOT included, we will need to add some words around the battery mon params that they would need adjustment for other power modules....finally, we should note which UARTs have DMA

**Comment by Cedric0489** [comment]
> @Cedric0489 need to add to common-autopilots.rst also, if the power module is NOT included, we will need to add some words around the battery mon params that they would need adjustment for other power modules....finally, we should note which UARTs have DMA
power module is included in this flight control. I have added PixPilot-v6 to the common-autopilots.rst and noted uarts dma-enabled.

File: common/source/docs/common-makeflyeasy-PixPilot-V6.rst [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+ - SERIAL0 -> console (primary mavlink, usually USB)
+ - SERIAL1 -> USART2  (telem1,DMA-enabled)
+ - SERIAL2 -> USART3  (Telem2,DMA-enabled)
+ - SERIAL3 -> UART4   (GPS1)
+ - SERIAL4 -> UART8   (GPS2)
+ - SERIAL5 -> UART7   (USER,DMA-enabled)
+
+ Connector pin assignments
+=========================
```
Comment (Hwurzburg) [inline common/source/docs/common-makeflyeasy-PixPilot-V6.rst:67]: remove indent before "Connector"

File: common/source/docs/common-makeflyeasy-PixPilot-V6.rst [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+I2C1, I2C2 ports
+---------------
```
Comment (Hwurzburg) [inline common/source/docs/common-makeflyeasy-PixPilot-V6.rst:105]: underline too short

File: common/source/docs/common-makeflyeasy-PixPilot-V6.rst [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+Power1, Power2 ports
```
Comment (Hwurzburg) [inline common/source/docs/common-makeflyeasy-PixPilot-V6.rst:298]: need a space about this line

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
a few build errors have come in

**Comment by Hwurzburg** [comment]
thanks..merging

**Pull request merged**